src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -40,7 +40,8 @@
 import com.amazonaws.services.securitytoken.model.AssumeRoleResult;
 import com.cloudbees.plugins.credentials.CredentialsDescriptor;
 import com.cloudbees.plugins.credentials.CredentialsScope;
-import edu.umd.cs.findbugs.annotations.CheckForNull;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
```

Comment:
The `findbugs` annotation (even if deprecated, will be fixed by upgrading the parent pom and using `spotbugs`) are preferred for license issues.